packages/core/src/profile/link.ts

```diff
@@ -0,0 +1,86 @@
+import { parseKey } from '@preevy/common'
+import * as jose from 'jose'
+import crypto from 'crypto'
+import fetch from 'node-fetch'
+import { TokenExpiredError, TokesFileSchema, getTokens } from '../login'
+import { profileStore } from './store'
+import { Store } from '../store'
+import { Logger } from '../log'
+
+export type Org = {id: string; name: string; role: string}
+
+export const link = async (
+  store: Store,
+  dataDir: string,
+  lcUrl: string,
+  logger: Logger,
+  promptUserWithChooseOrg: (orgs: Org[]) => Promise<Org>
+) => {
+  let tokens: TokesFileSchema | undefined
+  try {
+    tokens = await getTokens(dataDir)
+  } catch (e) {
+    if (e instanceof TokenExpiredError) {
+      logger.info('Session is expired, please log in again')
```

Comment:
this should be an error so the cli would exit with nonzero code

Reply:
why?